client/lib/plugins/wporg-data/actions.js

```diff
@@ -54,6 +57,18 @@ var PluginsDataActions = {
 			searchTerm: searchTerm
 		} );
 
+		if ( 'featured' === category ) {
+			this.fetchCuratedList( 'featured', [
+				'woocommerce',
+				'jetpack',
+				'bbpress',
+				'buddypress',
+				'akismet',
+				'vaultpress',
```

Comment:
True, good point. And if we need to support multiple lists, it would make sense to do other refactoring/Redux converting anyways.